src/main/java/StringCalculator/view/CalculatorView.java

```diff
@@ -0,0 +1,21 @@
+package StringCalculator.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class CalculatorView {
+
+    private final static Pattern SPACE = Pattern.compile(" ");
```

Comment:
상수 선언 시, 컨벤션에 따라 `static final` 순으로 작성해보면 좋을 것 같아요.
```suggestion
    private static final Pattern SPACE = Pattern.compile(" ");
```